components/Grid.js

```diff
@@ -5,9 +5,9 @@
 
 import propTypes from '@styled-system/prop-types';
 import styled from 'styled-components';
-import { color, compose, flexbox, grid, layout, space, typography } from 'styled-system';
+import { border, color, compose, flexbox, grid, layout, space, typography } from 'styled-system';
 
-const boxProps = compose(space, color, layout, typography, flexbox);
+const boxProps = compose(space, border, color, layout, typography, flexbox);
```

Comment:
I'd rather keep `Box` as simple as possible (mostly for performances reasons). Can we rather use the `Container` component - whose purpose is to have all the fancy props for customization?

Reply:
Makes sense, I'll roll back this.